Components/card/card.css

```diff
@@ -0,0 +1,209 @@
+
+
+/*=============================================
+=            Card with Badge            =
+=============================================*/
+
+.card-badge {
+    display: flex;
+    flex-flow: column wrap;
+    background: white;
+    flex-basis: 18.75rem;
+    margin: 1rem 0;
+    align-self: flex-start;
+    cursor: pointer;
+    position: relative;
+}
+
+.card-header {
+    display: flex;
+    flex-flow: column wrap;
+    text-align: left;
+}
+
+.card-header-txt {
+    padding: 0 1rem;
+}
+
+.card-header-txt h3{
+    margin-top: 0.5rem;
+    margin-bottom: 0;
+}
+
+.card-btn {
+    margin: 0.5rem;
+}
+
+.card-btn:hover {
+    background-color: var(--primary-color-shade);
+    cursor: pointer;
+    transition: all 0.3s;
+}
+
+.badge-in-card {
+    position: absolute;
+    top: 7px;
+    left: -2px;
+    background-color: var(--primary-color-shade);
+    font-size: 0.8rem;
+    font-weight: 600;
+    color: white;
+    padding: 0.1rem 0.3rem;
+    width: fit-content;
+}
+
+.card-floating-icon {
+    position: absolute;
+    top: 4px;
+    right: 2px;
+    font-size: 1.2rem;
+    border-radius: 50%;
+    padding: 0.3rem;
+}
+
+.card-floating-icon:hover {
+    background-color: var(--black-light-shade);
+}
+
+/*=============================================
+=            Card with Text Overlay            =
+=============================================*/
+
+.card-txt-overlay {
+    position: relative;
+    box-shadow: none;
+}
+
+.card-txt-overlay .badge-in-card {
+    display: none;
+}
+
+.overlay-msg {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    background-color: var(--off-white);
+    opacity: 75%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+
+/*=============================================
+=            Horizontal Card            =
+=============================================*/
+
+.card-horizontal {
+    display: flex;
+    flex-direction: row;
+    background: white;
+    width: 100%;
+    max-width: 31.25rem;
+    margin: 1rem 0;
+    border-radius: 5px;
+    align-self: flex-start;
+    cursor: pointer;
+}
+
+.card-horizontal .img-in-horizontal img{
```

Comment:
Use universal selector here in place of `img` tag.

Reply:
Made the required changes! ✨